admin_panel/exams/model.py

```diff
@@ -0,0 +1,84 @@
+from django.db import models
+from django.core.validators import MinValueValidator
+
+from admin_panel.organizations.model import Organization
+from utils.timestamp_mixin import TimestampMixin
+
+
+class Exam(TimestampMixin):
+    """
+    Core Exam model containing only the basic exam information.
+    Settings are managed at the batch level.
+    """
+    DIFFICULTY_CHOICES = [
+        ('Easy', 'Easy'),
+        ('Medium', 'Medium'),
+        ('Hard', 'Hard'),
+    ]
+
+    OBJECTIVE_CHOICES = [
+        ('Semester Exam', 'Semester Exam'),
+        ('Mock Exam', 'Mock Exam'),
+        ('Entrance Exam', 'Entrance Exam'),
+        ('Test Series', 'Test Series'),
+    ]
+
+    # Basic Details
+    name = models.CharField(
+        max_length=255,
+        help_text="Name of the exam"
+    )
+    objective = models.CharField(
+        max_length=50,
+        choices=OBJECTIVE_CHOICES,
+        help_text="Purpose of the exam"
+    )
+    duration = models.DurationField(
+        help_text="Duration of the exam"
+    )
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider using minutes for exam duration</b></div>
<div id="fix">

Consider using `PositiveIntegerField` instead of `DurationField` for exam duration to store minutes directly, which is more practical for exam timing management. This would also be consistent with common exam duration representations.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -36,6 +36,8 @@
 -    duration = models.DurationField(
 -        help_text="Duration of the exam"
 -    )
 +    duration = models.PositiveIntegerField(
 +        help_text="Duration of the exam in minutes",
 +        validators=[MinValueValidator(1)]
 +    )
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/sleep-404/ExamEvoke/pull/8#issuecomment-2499580485>#1f7607</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged